espnet2/enh/layers/beamformer.py

```diff
@@ -9,19 +10,86 @@
 from torch_complex.tensor import ComplexTensor
 
 EPS = torch.finfo(torch.double).eps
+is_torch_1_8_plus = LooseVersion(torch.__version__) >= LooseVersion("1.8.0")
 
 
-def complex_norm(c: ComplexTensor) -> torch.Tensor:
+def is_complex(c):
```

Comment:
I think these functions can live in a utility module with `is_torch_1_8_plus` logic.

Reply:
Thanks, I have added the condition and move the related functions to a new script `complex_utils.py`.